dev-python/autobahn/autobahn-0.16.0.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+PYTHON_COMPAT=( python2_7 python3_4 )
```

Comment:
No 3.5 because many of the new deps were not tested with 3.5 yet.  If upstream did testing with 3.5 I tried to keep 3.5 in, but I had to remove it in some because of  deps that were not properly tested with 3.5.  And I can't rely on downstream testing in 3.5 to add 3.5 upstream.  I will do some more testing and keywording/3.5 adding in the coming weeks.  But I needed them in the tree sooner rather than later.

I was under the impression that we should not add 3.3 for new pkgs as 3.3 is nearing masking for removal.  I had to remove 3.3 from this one due to new dependencies.  I will  try to add 3.3 if possible when doing the 3.5 testing for all the deps.

In the meantime, py compat was restricted to what they are due to repoman complaining.